common.js

```diff
@@ -1621,6 +1621,65 @@ const allSoloCars = {
       'all': ['stx', 'fsp'],
     },
   },
+  'Ferrari': {
+    '250 (non-LM)': {
+      'all': ['asp'],
+    },
+    '275': {
+      'all': ['asp'],
+    },
+    '308 Coupe & Spider': {
+      'all': ['asp'],
+    },
+    '308 & 328': {
```

Comment:
You have `308 & 328` here and then `308 (all)` later. I think it makes sense to have separate 308 and 328 models, and then add the `cs` and `fp` classes to 308 while keeping only `cs` for 328.

Reply:
Yes you are right, will do asap